charts/stable/jellyfin/values.yaml

```diff
@@ -2,12 +2,24 @@ image:
   repository: tccr.io/truecharts/jellyfin
   pullPolicy: IfNotPresent
   tag: v10.8.10@sha256:d2c377ee7ea463110a1dda7eb1b231424ad05245aaf95744e76164bd2e593377
+broadcastProxyImage:
+  repository: alpine/socat
+  pullPolicy: IfNotPresent
+  tag: 1.7.4.4@sha256:8c1b44c58dcc6a63aa231c14b4e7fc237eec6d3b93cc0c88538fa4edba3921f9
```

Comment:
Can we move this to tccr please?

Reply:
Hi, started adding it but I noticed there is already a socat image there:
https://github.com/truecharts/containers/tree/master/mirror/socat

However, it is quite stale (not been updated in about a year). I also don't see it in the released images?
Moreover, its base image on Docker Hub is 7 years old. Should I file a PR that replaces the base image with the one I used in this PR (alpine/socat)?

Reply:
You can safely replace it afaik...

Reply:
Opened a PR - https://github.com/truecharts/containers/pull/23778

Reply:
Merged btw, once that's added I think we're safe to merge this :)

Reply:
Our containers aren't backed by ghcr at all.